docs/guides/vault/address-restrict-to-vault.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
file name:

`address-restrict-to-vault.md`

or

`restrict-addresses-to-vaults.md` ?

Let's keep it as close as possible to the article's title